app/javascript/components/sessions/SignIn.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+
+async function createSession(sessionUser) {
+  const sessionUrl = '/api/v1/sessions';
+
+  await fetch(sessionUrl, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      status: 'success',
+      user: {
+        email: sessionUser.email,
+        password: sessionUser.password,
+      },
+    }),
+  })
+    .then((response) => {
+      if (response.status === 204) {
+        // TODO: samuel - Need to discuss how to Authenticate the user
+        // Do Something
+      } else {
+        // Do Something
+      }
+    })
+    .catch((error) => {
+      console.log('json response is error', error);
+    });
+}
+
+export default function SignIn() {
+  type FormInputs = {
+    email: string;
+    password: string;
+    auth: boolean;
+  };
+
+  const { register, handleSubmit, formState: { errors } } = useForm<FormInputs>();
+
+  const handleSignIn = (sessionUser) => createSession(sessionUser);
+
+  return (
+    <div className="col-md-3 mx-auto">
+      <h1> Sign In </h1>
+      <Form onSubmit={handleSubmit(handleSignIn)}>
+        <Form.Group className="mb-3" controlId="signInEmail">
+          <Form.Label>Email:</Form.Label>
+          <Form.Control
+            type="email"
+            placeholder="Email"
+            {...register('email', { required: 'Email is required.' })}
+          />
+          <Form.Text className="text-danger d-block">
```

Comment:
That should do it.
Also, we can leverage the ` <Form.Control.Feedback type="invalid">` component.